starter-kits/credential-registry/docker/manage

```diff
@@ -277,6 +279,13 @@ build-api() {
     '../server/tob-api' \
     "$BASE_IMAGE" \
     'django'
+  
+  # this is only required in Windows, but it won't hurt to run it regardless of the platform
```

Comment:
I'd recommend wrapping the code in an `if` so it is only run when executed on a Windows system;

```
if [[ "$OSTYPE" == "msys" ]]; then
  echo -e "\nFixing file permissions in final tob-api (django) image"
  docker build -q \
    -t 'django' \
    -f '../../../credential-registry/server/Dockerfile' '../../../credential-registry/server'
fi
```

Reply:
I had deliberately left it out since there could in theory be more cases to handle (e.g.: `cygwin`) that could get "masked" this way, but if it seems better I'll add the check.